### ISSUE TITLE
Feature Request: Ability to call Windows Terminal through Win+X Quick Link Menu

### ISSUE THREAD

**Issue by Aster-the-Med-Stu** [microsoft/terminal#1132]
# Summary of the new feature/enhancement

Apparently the new Windows Terminal would be some people's default terminal. It is possible that they wish to replace PowerShell/cmd in Quick Link Menu with it. Currently there are 3rd softwares/scripts doing the job, but they couldn't add keyboard shortcut. So please add an option to add it to Quick Link menu in Windows Terminal's settings.

**Comment by hez2010** [comment]
Currently you can simply run 'wtd' to start Windows Terminal quickly.

**Comment by zadjii-msft** [comment]
This is actually something that'll be fixed by #492, whenever we end up adding support for that. When we do add that feature, then when you open a commandline application, it will launch in your default terminal application, as opposed to conhost. So using Win+X to launch you shell will actually launch the shell attached to Windows Terminal.

**Comment by mottosso** [comment]
> Currently you can simply run 'wtd' to start Windows Terminal quickly.

Is this still supposed to work? Neither `wt` or `wtd` is available here. I've got the Preview build, released yesterday on Windows Store. In my Windows Terminal application directory, I've got `conhost.exe` and `WindowsTerminal.exe`, however this directory wasn't added to my PATH on install, and I'm not sure if it should.

```bash
# Install dir
C:\Program Files\WindowsApps\Microsoft.WindowsTerminal_0.2.1715.0_x64__8wekyb3d8bbwe
```
